.github/actions/run-playwright/action.yml

```diff
@@ -33,13 +33,18 @@ runs:
     - name: Get dashboard address
       if: ${{ inputs.run == 'true' }}
       id: dashboard
-      run: echo "address=http://$(address=$(juju show-unit dashboard/0 | yq '.dashboard/0.public-address'); if [ "$address" != "null" ]; then echo $address; else juju show-unit dashboard/0 | yq '.dashboard/0.address'; fi):8080" >> $GITHUB_OUTPUT
+      run: |
+        if [[ -z "${DASHBOARD_ADDRESS}" ]]; then
+          echo "address=http://$(address=$(juju show-unit dashboard/0 | yq '.dashboard/0.public-address'); if [ "$address" != "null" ]; then echo $address; else juju show-unit dashboard/0 | yq '.dashboard/0.address'; fi):8080" >> $GITHUB_OUTPUT
+        else
+          echo "address=http://$DASHBOARD_ADDRESS" >> $GITHUB_OUTPUT
```

Comment:
This allows it to use the provided dashboard address as there is no juju unit to get an address from.